spectre-meltdown-checker.sh

```diff
@@ -1217,10 +1198,23 @@ check_variant3()
 	elif [ -z "$msg" ]; then
 		# if msg is empty, sysfs check didn't fill it, rely on our own test
 		if [ "$opt_live" = 1 ]; then
+			# Test if the current host is a Xen PV Dom0 / DomU
+			dmesg_grep 'Booting paravirtualized kernel on Xen$'; ret=$?
+			if [ $ret -eq 2 ]; then
+				_warn "dmesg truncated, please reboot and relaunch this script"
```

Comment:
I'd like to avoid warning users when their dmesg is truncated and they're not running Xen at all.
Maybe add a generic Xen check (regardless of domU or dom0) around all this Xen-specific stuff?

Reply:
Can do, that is what I wanted to do at first actually, ended up going the lazy way :)